client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -185,7 +161,33 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 			subHeaderText={ headerDescription }
 			fallbackSubHeaderText={ headerDescription }
 			align={ isReskinned ? 'left' : 'center' }
-			stepContent={ getContent() }
+			stepContent={
+				<>
+					<div className="confirm__info-section" />
+					<div className="confirm__instructions-container">
+						{ getWPComSubdomainWarningContent() }
+						{ getCheckoutContent() }
+						{ getWarningsOrHoldsSection() }
+						<ActionSection>
+							<SupportLink />
+							<StyledNextButton
+								disabled={ hasBlockers || ! isDataReady }
+								onClick={ () => {
+									dispatch(
+										submitSignupStep( { stepName: 'confirm' }, { siteConfirmed: siteId } )
+									);
+									if ( siteUpgrading.required ) {
+										return ( window.location.href = siteUpgrading.checkoutUrl );
```

Comment:
This seems to work but please test with a few attempts, can this redirect break the dispatch or is that all in sync?

One of my tests got a checkout errorNotice complaining about the plan selection being undefined. Guessing maybe old site not compatible with the "business-bundle" plan selection?